include/seqan3/alignment/pairwise/edit_distance_fwd.hpp

```diff
@@ -61,6 +61,16 @@ template <std::ranges::viewable_range database_t,
           typename word_t = uint_fast64_t>
 struct default_edit_distance_trait_type
 {
+    //!\brief The type of the alignment config.
+    using align_config_type = std::remove_reference_t<align_config_t>;
+
+private:
+    //!\brief The type score in the seqan3::alignment_result object (may be different from the internally used type).
+    using result_score_type = std::conditional_t<align_config_type::template exists<align_cfg::result>(),
+                                                 typename std::remove_reference_t<
+                                                     decltype(get<align_cfg::result>(align_config_type{}))>::score_type,
+                                                 typename align_cfg::result<>::score_type>;
+public:
     //!\brief The type of one machine word.
     using word_type = word_t;
     static_assert(std::is_unsigned_v<word_type>, "the word type of edit_distance_unbanded must be unsigned.");
```

Comment:
`\/ \/ \/ \/ \/ \/ \/ \/ \/ \/ \/` score_type definition - below this `\/ \/ \/ \/ \/ \/ \/ \/ \/ \/ \/`